packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,6 +6,23 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## July
 
+### Weekly release 2022.07.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Dynamic issuer mode for identity providers](#dynamic-issuer-mode-for-identity-providers)                    | July 13, 2022            |
+| [Bugs fixed in 2022.07.1](#bugs-fixed-in-2022-07-1)                        | July 13, 2022            |
+
+#### Dynamic issuer mode for identity providers
+
+You can configure the dynamic issuer mode for an identity provider using the Identity Provider API. When you set the`issuerMode` parameter to `DYNAMIC`, Okta uses the domain from the Authorize URL as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```

Comment:
```suggestion
You can configure the dynamic issuer mode for an Identity Provider using the Identity Provider API. When you set the`issuerMode` parameter to `DYNAMIC`, Okta uses the domain from the Authorize URL as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```
Sorry, I missed this earlier. Our capitalization guide says to capitalize Identity Provider.